R/run.R

```diff
@@ -0,0 +1,150 @@
+#' Run multiple Stock Synthesis models
+#'
+#' Loops over a vector of directories and iteratively runs SS in each one
```

Comment:
Change SS to SS3

Reply:
Good catch. I revised the whole paragraph in addition to making this change.